sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/csv/UnivocityParserSuite.scala

```diff
@@ -303,20 +303,6 @@ class UnivocityParserSuite extends SparkFunSuite with SQLHelper {
       requiredSchema = StructType.fromDDL("i INTEGER, d DOUBLE"),
       filters = Seq(EqualTo("d", 3.14)),
       expected = Some(InternalRow(1, 3.14)))
-
-    val errMsg = intercept[IllegalArgumentException] {
-      check(filters = Seq(EqualTo("invalid attr", 1)), expected = None)
-    }.getMessage
-    assert(errMsg.contains("invalid attr does not exist"))
```

Comment:
Hi, @MaxGekk . Sorry, but could you tell me where this test coverage is moved to by this PR?

Reply:
I don't remember already why I removed the tests half year ago. Let me try to add them back. I would guess because they are useless since filters w/ invalid attrs must not pushed down at all.